gst/nnstreamer/tensor_transform/tensor_transform.c

```diff
@@ -765,7 +765,13 @@ gst_tensor_transform_set_option_data (GstTensorTransform * filter)
     case GTT_TYPECAST:
     {
       if (g_regex_match_simple (REGEX_TYPECAST_OPTION, filter->option, 0, 0)) {
-        filter->data_typecast.to = gst_tensor_get_type (filter->option);
+        tensor_type type_to = gst_tensor_get_type (filter->option);
+#ifdef HAVE_ORC
+        if (type_to == _NNS_INT64 || type_to == _NNS_UINT64) {
+          filter->orc_supported = FALSE;
+        }
```

Comment:
need ```filter->orc_supported = TRUE;``` somewhere in case type_to has been changed from INT64 to INT8 (or others).

Reply:
Right!
I changed to decide `orc_supported` during the negotiation process.
It covers these cases.